### ISSUE TITLE
Unable to use Maven snapshot

### ISSUE THREAD

**Issue by buremba** [duckdb/duckdb#8240]
### What happens?

I get the following exception in Gradle:

```
> Could not find duckdb_jdbc-0.9.0-SNAPSHOT.jar (org.duckdb:duckdb_jdbc:0.9.0-SNAPSHOT:20230713.020012-165).
     Searched in the following locations:
         https://oss.sonatype.org/content/repositories/snapshots/org/duckdb/duckdb_jdbc/0.9.0-SNAPSHOT/duckdb_jdbc-0.9.0-20230713.020012-165.jar

```

These are the files for version 165 listed [in the Maven repo](https://oss.sonatype.org/content/repositories/snapshots/org/duckdb/duckdb_jdbc/0.9.0-SNAPSHOT/):

<img width="806" alt="CleanShot 2023-07-13 at 17 16 39@2x" src="https://github.com/duckdb/duckdb/assets/82745/232c95a3-341d-44f3-a33b-77ab609d74cb">

The jar exists for the previous versions, specifically for 164.

It seems like the [maven-metadata.xml](https://oss.sonatype.org/content/repositories/snapshots/org/duckdb/duckdb_jdbc/0.9.0-SNAPSHOT/maven-metadata.xml
) points to files that doesn't exist:

<img width="456" alt="CleanShot 2023-07-13 at 17 18 07@2x" src="https://github.com/duckdb/duckdb/assets/82745/c3d13441-e0a2-435a-91cc-114b7df1d86b">

### To Reproduce

Follow the instructions on the installation page for GitHub Master: https://duckdb.org/docs/installation/?environment=java

### OS:

aarch64

### DuckDB Version:

0.9.0-SNAPSHOT

### DuckDB Client:

Java

### Full Name:

Burak Emre Kabakci

### Affiliation:

LiveRamp

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by buremba** [comment]
Action doesn't seem to output anything useful: https://github.com/duckdb/duckdb/actions/runs/5542730551/jobs/10119923571

**Comment by buremba** [comment]
There is even a check in the code for the file existence so I'm not sure what the issue is: https://github.com/duckdb/duckdb/blob/0e0fd210cde56142e4c4630bd529d45fb9740480/scripts/jdbc_maven_deploy.py#L147

**Comment by jonathanswenson** [comment]
This does not appear to be working for me either. Failing with a similar error: 
```
> Could not resolve all files for configuration ':XXX:runtimeClasspath'.
   > Could not find duckdb_jdbc-0.9.0-SNAPSHOT.jar (org.duckdb:duckdb_jdbc:0.9.0-SNAPSHOT:20230918.020827-363).
     Searched in the following locations:
         https://oss.sonatype.org/content/repositories/snapshots/org/duckdb/duckdb_jdbc/0.9.0-SNAPSHOT/duckdb_jdbc-0.9.0-20230918.020827-363.jar
```

**Comment by Mause** [comment]
Sounds like this is a Gradle specific issue then?

**Comment by jonathanswenson** [comment]
Yeah that sounds right. A test maven config (generated from `maven-archetype-quickstart` and added the repo / dependencies as specified in https://duckdb.org/docs/installation/?environment=java) seems to work fine.

**Comment by Mause** [comment]
It's possible the underlying issue is https://github.com/gradle/gradle/issues/15095

**Comment by Mause** [comment]
First attempt to fix this didn't quite work, needs more testing

**Comment by brianwyka** [comment]
Looks like the most recent `SNAPSHOT` has all the `.asc`, `.sha1`, and `.md5` files, but is missing the actual `.pom` and `.jar` files.
https://oss.sonatype.org/content/repositories/snapshots/org/duckdb/duckdb_jdbc/0.10.0-SNAPSHOT/

Hope this helps...

**Comment by hannes** [comment]
That those specific versions are not there seems to be a largely academic problem - not all nightly runs are successful. The last successful snapshot Jar seems to be this one:
https://oss.sonatype.org/content/repositories/snapshots/org/duckdb/duckdb_jdbc/0.11.0-SNAPSHOT/duckdb_jdbc-0.11.0-20240311.011140-79.jar
Uploaded yesterday - so all well from my vantage point.

**Comment by Mause** [comment]
I think you've misunderstood the issue here Hannes - it isn't that specific versions are missing, but that the version files that are pointed to in the maven metadata file are missing

**Comment by buremba** [comment]
@Mause is right on spot here. The issue still exists in master.

**Comment by staticlibs** [comment]
Maven snapshots now work again, fixed in duckdb/duckdb-java#546, usage example (version to be updated):

```xml
        <dependency>
            <groupId>org.duckdb</groupId>
            <artifactId>duckdb_jdbc</artifactId>>
            <version>1.4.5.0-2b2af072-SNAPSHOT</version>
        </dependency>
```
```xml
        <repository>
            <id>central-snapshots</id>
            <url>https://central.sonatype.com/repository/maven-snapshots/</url>
            <snapshots>
                <enabled>true</enabled>
            </snapshots>
        </repository>
```
